### DISCUSSION TITLE
Trigger synchronizations

### DISCUSSION THREAD

**Discussion by andreaslimidev** [inngest/inngest#1072] [Ideas]
I have a suggestion for a feature regarding synchronisations. We are using Vercel, and had to use the instant rollback feature today to revert to a previous commit in production. We are also using Inngest, of course, which we noticed did not synchronise with the rollback deployment. 

To remedy this, we pushed a new commit to production in order to trigger the Inngest synch. We were now able to rerun the failed function which fortunately used the new synch. 

My idea for a feature would be to manually trigger synchronisations from the Inngest dashboard. Would love to hear your thoughts around this.

**Comment by tonyhb** [comment]
Added — thank you so much for the feedback!
